address/address_test.go

```diff
@@ -308,3 +309,71 @@ func TestAddressEncoding(t *testing.T) {
 		}
 	}
 }
+
+// TestBIPTestVectors tests that the BIP test vectors are passing.
+func TestBIPTestVectors(t *testing.T) {
+	t.Parallel()
+
+	testVectors := &TestVectors{}
+	test.ParseTestVectors(
+		t, "testdata/address_tlv_encoding.json", &testVectors,
```

Comment:
I think I'm missing how/where the test vectors were _generated_ in the first place. Eg: would expect an optional flag on an existing (?) test to output the vectors in a specified location.

Reply:
Good point! I went ahead and added a build flag (can be activated using `make unit gen-test-vectors=true`) to generate the files and also invested some time to make the content deterministic. So it should be as easy as adding new test cases in Golang to generate extended test vectors!